crates/gitbutler-tauri/Cargo.toml

```diff
@@ -63,13 +63,18 @@ features = [
     "window-unmaximize"
 ]
 
+[target."cfg(windows)".features]
```

Comment:
Do you mean `dependencies` here?

Reply:
That's a great catch! Actually I meant to change the default feature based on the platform, but that doesn't work (yet).
Hence I think the only way to do this is to explicitly enable the feature, which I hope I have done for `release.sh` and for CI.

Reply:
> Actually I meant to change the default feature based on the platform

Yeah 🙄 I've been waiting for Cargo to support some of this for _years_ for my OS project. It's a huge PITA.